api-lib/s3.ts

```diff
@@ -0,0 +1,30 @@
+import aws from "aws-sdk";
+
+const s3 = new aws.S3({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID || 'missing_env_var',
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || 'missing_env_var',
+  endpoint: process.env.AWS_ENDPOINT ? process.env.AWS_ENDPOINT : undefined,
+});
+
+const uploadImage = (Key: string, Body: Buffer) => {
+  const p = {
```

Comment:
nit: can we give this a name? a proper name is much more information-rich, even if the variable is trivial than a letter.

Reply:
done